### PR TITLE
docs: add 'Open in StackBlitz' buttons to various examples

### PR DESCRIPTION
**Pull request by d3lm**
This PR adds a `Preview` section and a `Open in StackBlitz` button to various examples. I have tested all examples and omitted the ones that require third party API keys, or didn't work. Some examples don't work locally either.

Here's an example:
image

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [x] Examples updated
- [ ] Telemetry added. In case of a feature if it's used or not.

## Documentation / Examples

- [x] Make sure the linting passes

### CODE REVIEW

File: examples/analyze-bundles/package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "analyze": "cross-env ANALYZE=true yarn build"
+    "analyze": "cross-env ANALYZE=true yarn build",
+    "serve": "serve .next/analyze"
```
Comment (d3lm) [inline examples/analyze-bundles/package.json:9]: I have added a `serve` script so that you can easily serve the output of the analyzer.

**Review by leerob — APPROVED** [review]

**Comment by leerob** [comment]
Would you mind adding this list of broken examples to a new GH issue? 🙏

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| buildDuration | 11.1s | 11.4s | ⚠️ +229ms |
| buildDurationCached | 2.8s | 2.8s | ⚠️ +19ms |
| nodeModulesSize | 46.7 MB | 46.7 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.288 | 2.412 | ⚠️ +0.12 |
| / avg req/sec | 1092.44 | 1036.54 | ⚠️ -55.9 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.168 | 1.239 | ⚠️ +0.07 |
| /error-in-render avg req/sec | 2140.86 | 2018.26 | ⚠️ -122.6 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| framework-HASH.js gzip | 39.3 kB | 39.3 kB | ✓ |
| main-HASH.js gzip | 20.2 kB | 20.2 kB | ✓ |
| webpack-HASH.js gzip | 804 B | 804 B | ✓ |
| Overall change | 60.3 kB | 60.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _app-HASH.js gzip | 801 B | 801 B | ✓ |
| _error-HASH.js gzip | 3.07 kB | 3.07 kB | ✓ |
| amp-HASH.js gzip | 527 B | 527 B | ✓ |
| css-HASH.js gzip | 334 B | 334 B | ✓ |
| hooks-HASH.js gzip | 890 B | 890 B | ✓ |
| index-HASH.js gzip | 263 B | 263 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 333 B | 333 B | ✓ |
| withRouter-HASH.js gzip | 330 B | 330 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _buildManifest.js gzip | 391 B | 391 B | ✓ |
| Overall change | 391 B | 391 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| index.html gzip | 572 B | 572 B | ✓ |
| link.html gzip | 579 B | 579 B | ✓ |
| withRouter.html gzip | 567 B | 567 B | ✓ |
| Overall change | 1.72 kB | 1.72 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| buildDuration | 12.4s | 12.8s | ⚠️ +407ms |
| buildDurationCached | 3.8s | 3.9s | ⚠️ +109ms |
| nodeModulesSize | 46.7 MB | 46.7 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| framework-HASH.js gzip | 39.3 kB | 39.3 kB | ✓ |
| main-HASH.js gzip | 20.2 kB | 20.2 kB | ✓ |
| webpack-HASH.js gzip | 804 B | 804 B | ✓ |
| Overall change | 60.3 kB | 60.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _app-HASH.js gzip | 801 B | 801 B | ✓ |
| _error-HASH.js gzip | 3.07 kB | 3.07 kB | ✓ |
| amp-HASH.js gzip | 527 B | 527 B | ✓ |
| css-HASH.js gzip | 334 B | 334 B | ✓ |
| hooks-HASH.js gzip | 890 B | 890 B | ✓ |
| index-HASH.js gzip | 263 B | 263 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 333 B | 333 B | ✓ |
| withRouter-HASH.js gzip | 330 B | 330 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _buildManifest.js gzip | 391 B | 391 B | ✓ |
| Overall change | 391 B | 391 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _error.js | 16.9 kB | 16.9 kB | ✓ |
| 404.html | 2.44 kB | 2.44 kB | ✓ |
| 500.html | 2.43 kB | 2.43 kB | ✓ |
| amp.amp.html | 10.8 kB | 10.8 kB | ✓ |
| amp.html | 1.63 kB | 1.63 kB | ✓ |
| css.html | 1.81 kB | 1.81 kB | ✓ |
| hooks.html | 1.69 kB | 1.69 kB | ✓ |
| index.js | 17.2 kB | 17.2 kB | ✓ |
| link.js | 17.4 kB | 17.4 kB | ✓ |
| routerDirect.js | 17.4 kB | 17.4 kB | ✓ |
| withRouter.js | 17.4 kB | 17.4 kB | ✓ |
| Overall change | 107 kB | 107 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| buildDuration | 9.9s | 9.9s | -48ms |
| buildDurationCached | 4s | 4.2s | ⚠️ +155ms |
| nodeModulesSize | 46.7 MB | 46.7 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.261 | 2.313 | ⚠️ +0.05 |
| / avg req/sec | 1105.49 | 1081.04 | ⚠️ -24.45 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.16 | 1.145 | -0.01 |
| /error-in-render avg req/sec | 2155.69 | 2182.97 | +27.28 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| 677f882d2ed8..HASH.js gzip | 13.3 kB | 13.3 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-HASH.js gzip | 7.99 kB | 7.99 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| Overall change | 61.1 kB | 61.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _app-HASH.js gzip | 1.07 kB | 1.07 kB | ✓ |
| _error-HASH.js gzip | 3.74 kB | 3.74 kB | ✓ |
| amp-HASH.js gzip | 536 B | 536 B | ✓ |
| css-HASH.js gzip | 339 B | 339 B | ✓ |
| hooks-HASH.js gzip | 887 B | 887 B | ✓ |
| index-HASH.js gzip | 227 B | 227 B | ✓ |
| link-HASH.js gzip | 1.63 kB | 1.63 kB | ✓ |
| routerDirect..HASH.js gzip | 303 B | 303 B | ✓ |
| withRouter-HASH.js gzip | 302 B | 302 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 9.16 kB | 9.16 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _buildManifest.js gzip | 420 B | 420 B | ✓ |
| Overall change | 420 B | 420 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| index.html gzip | 627 B | 627 B | ✓ |
| link.html gzip | 633 B | 633 B | ✓ |
| withRouter.html gzip | 620 B | 620 B | ✓ |
| Overall change | 1.88 kB | 1.88 kB | ✓ |

</details>

</details>
Commit: 89eb5e0e1c46f9cc222ec0a0a3f81b5050bb0292

**Comment by d3lm** [comment]
Yep totally, I ll extract them and file a new issue with the examples that don't work.

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 89eb5e0e1c46f9cc222ec0a0a3f81b5050bb0292

**test/integration/rewrite-with-browser-history/test/index.test.js** 
- rewrites persist with browser history actions > dev mode > back-button should go back to rewritten path successfully

<details>
<summary>Expand output</summary>

  ● rewrites persist with browser history actions › dev mode › back-button should go back to rewritten path successfully

    NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":"#another"}
      (Session info: headless chrome=91.0.4472.77)

      33 |     await browser.back()
      34 |
    > 35 |     expect(await browser.elementByCss('#another').text()).toBe('another page')
         |            ^
      36 |
      37 |     expect(await browser.eval('window.beforeNav')).toBe(1)
      38 |   })

      at Object.throwDecodedError (../node_modules/selenium-webdriver/lib/error.js:550:15)
      at parseHttpResponse (../node_modules/selenium-webdriver/lib/http.js:565:13)
      at Executor.execute (../node_modules/selenium-webdriver/lib/http.js:491:26)
      at thenableWebDriverProxy.execute (../node_modules/selenium-webdriver/lib/webdriver.js:700:17)
      at Object.<anonymous> (integration/rewrite-with-browser-history/test/index.test.js:35:12)

</details>

**Review by ijjk — APPROVED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| buildDuration | 12.9s | 12.8s | -105ms |
| buildDurationCached | 3.2s | 3.2s | -11ms |
| nodeModulesSize | 46.5 MB | 46.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.541 | 2.545 | 0 |
| / avg req/sec | 983.68 | 982.14 | ⚠️ -1.54 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.311 | 1.325 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 1907.54 | 1886.56 | ⚠️ -20.98 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| framework-HASH.js gzip | 39.3 kB | 39.3 kB | ✓ |
| main-HASH.js gzip | 20.2 kB | 20.2 kB | ✓ |
| webpack-HASH.js gzip | 804 B | 804 B | ✓ |
| Overall change | 60.3 kB | 60.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _app-HASH.js gzip | 801 B | 801 B | ✓ |
| _error-HASH.js gzip | 3.07 kB | 3.07 kB | ✓ |
| amp-HASH.js gzip | 527 B | 527 B | ✓ |
| css-HASH.js gzip | 334 B | 334 B | ✓ |
| hooks-HASH.js gzip | 890 B | 890 B | ✓ |
| index-HASH.js gzip | 263 B | 263 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 333 B | 333 B | ✓ |
| withRouter-HASH.js gzip | 330 B | 330 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _buildManifest.js gzip | 392 B | 392 B | ✓ |
| Overall change | 392 B | 392 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| index.html gzip | 573 B | 573 B | ✓ |
| link.html gzip | 580 B | 580 B | ✓ |
| withRouter.html gzip | 567 B | 567 B | ✓ |
| Overall change | 1.72 kB | 1.72 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| buildDuration | 14.6s | 14.7s | ⚠️ +129ms |
| buildDurationCached | 4.3s | 4.3s | -8ms |
| nodeModulesSize | 46.5 MB | 46.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| framework-HASH.js gzip | 39.3 kB | 39.3 kB | ✓ |
| main-HASH.js gzip | 20.2 kB | 20.2 kB | ✓ |
| webpack-HASH.js gzip | 804 B | 804 B | ✓ |
| Overall change | 60.3 kB | 60.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _app-HASH.js gzip | 801 B | 801 B | ✓ |
| _error-HASH.js gzip | 3.07 kB | 3.07 kB | ✓ |
| amp-HASH.js gzip | 527 B | 527 B | ✓ |
| css-HASH.js gzip | 334 B | 334 B | ✓ |
| hooks-HASH.js gzip | 890 B | 890 B | ✓ |
| index-HASH.js gzip | 263 B | 263 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 333 B | 333 B | ✓ |
| withRouter-HASH.js gzip | 330 B | 330 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _buildManifest.js gzip | 392 B | 392 B | ✓ |
| Overall change | 392 B | 392 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _error.js | 16.9 kB | 16.9 kB | ✓ |
| 404.html | 2.44 kB | 2.44 kB | ✓ |
| 500.html | 2.43 kB | 2.43 kB | ✓ |
| amp.amp.html | 10.8 kB | 10.8 kB | ✓ |
| amp.html | 1.63 kB | 1.63 kB | ✓ |
| css.html | 1.81 kB | 1.81 kB | ✓ |
| hooks.html | 1.69 kB | 1.69 kB | ✓ |
| index.js | 17.2 kB | 17.2 kB | ✓ |
| link.js | 17.4 kB | 17.4 kB | -2 B |
| routerDirect.js | 17.4 kB | 17.4 kB | ✓ |
| withRouter.js | 17.4 kB | 17.4 kB | ✓ |
| Overall change | 107 kB | 107 kB | -2 B |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| buildDuration | 11.5s | 11.3s | -203ms |
| buildDurationCached | 4.7s | 4.7s | ⚠️ +6ms |
| nodeModulesSize | 46.5 MB | 46.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.604 | 2.595 | -0.01 |
| / avg req/sec | 960.09 | 963.25 | +3.16 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.293 | 1.314 | ⚠️ +0.02 |
| /error-in-render avg req/sec | 1933.19 | 1902.58 | ⚠️ -30.61 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| 677f882d2ed8..HASH.js gzip | 13.4 kB | 13.4 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-HASH.js gzip | 7.99 kB | 7.99 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| Overall change | 61.1 kB | 61.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _app-HASH.js gzip | 1.07 kB | 1.07 kB | ✓ |
| _error-HASH.js gzip | 3.74 kB | 3.74 kB | ✓ |
| amp-HASH.js gzip | 536 B | 536 B | ✓ |
| css-HASH.js gzip | 339 B | 339 B | ✓ |
| hooks-HASH.js gzip | 887 B | 887 B | ✓ |
| index-HASH.js gzip | 227 B | 227 B | ✓ |
| link-HASH.js gzip | 1.63 kB | 1.63 kB | ✓ |
| routerDirect..HASH.js gzip | 303 B | 303 B | ✓ |
| withRouter-HASH.js gzip | 302 B | 302 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 9.17 kB | 9.17 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| _buildManifest.js gzip | 420 B | 420 B | ✓ |
| Overall change | 420 B | 420 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | d3lm/next.js feat/examples/open-in-stackblitz | Change |
| - | - | - | - |
| index.html gzip | 625 B | 625 B | ✓ |
| link.html gzip | 632 B | 632 B | ✓ |
| withRouter.html gzip | 620 B | 620 B | ✓ |
| Overall change | 1.88 kB | 1.88 kB | ✓ |

</details>

</details>
Commit: 6217c337146902d60fe93a6c54aa7b9633b78cdd

**Pull request merged**